query/src/main/scala/hmda/query/repository/filing/FilingComponent.scala

```diff
@@ -296,28 +299,79 @@ trait FilingComponent { this: DbConfiguration =>
     def deleteByRespondentId(respId: String) = db.run(table.filter(_.respondentId === respId).delete)
   }
 
-  class LarTotalTable(tag: Tag) extends Table[LoanApplicationRegisterTotal](tag, "lars_total") {
-    def id = column[Int]("id", O.PrimaryKey)
-    def respondentId = column[String]("respondent_id")
-    def period = column[String]("period")
-    def total = column[Int]("total")
-
-    override def * = (id, respondentId, period, total) <> (LoanApplicationRegisterTotal.tupled, LoanApplicationRegisterTotal.unapply)
+  class LarTotalTable(tag: Tag) extends Table[Msa](tag, "lars_total") {
```

Comment:
Not a major thing, but the naming is a bit off now imho, "lars_total" as a table name is mapped to `Msa`. I think the table name should reflect the fact that it mostly contains information by `msa`. 
Maybe `lars_total_msa`?